flight_declaration_operations/views.py

```diff
@@ -299,6 +298,16 @@ def set_flight_declaration(request: HttpRequest):
 @api_view(["POST"])
 @requires_scopes(["blender.write"])
 def set_signed_flight_declaration(request: HttpRequest):
+    try:

```

Comment:
Moved media type check to the top so that we can skip verifying the request if the content type is not `application/json`